### DISCUSSION TITLE
兄弟们你们的GPT还能正常使吗？我这一直所超出配额

### DISCUSSION THREAD

**Discussion by heiheiheibj** [ChatGPTNextWeb/NextChat#4527] [Q&A]
![2b8aec5e2f81ddbb2ab564b017a3424](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/6910198/4b6a5d13-9e0d-45f6-8fa8-2e6ac5d3801a)
![3fac9261d80b6080a058888824111d9](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/6910198/aa0d69b7-8218-416a-8699-f0d5841d4e9e)
我是买的会员，而且这个月基本就没用，刚才一用不好使了
不但4不好使，GPT3.5也是一个样，兄弟们有遇到过不

**Comment by iiicebearrr** [comment]
API要单独充值, 检查你的API额度还有没有

**Comment by 95256155o** [comment]
OPENAI改预付费了，现在需要先充值再用

**Comment by heiheiheibj** [comment]
谢谢几位，果然，还得单充值，搞的太麻烦了
